testCases/current/tearDown.sh

```diff
@@ -0,0 +1,4 @@
+cd ../testRepo
+
+git tag -d v1.1.1 > /dev/null 2>&1
```

Comment:
i do not like this approach-  i will rather copy  clean repo in it's place.
because whewn this test do unexpected (make another tag) next test's env will be unexpectally different and it could end in cascade fail

Reply:
Yes. I see your point